Makefile

```diff
@@ -73,17 +85,19 @@ gen-certs:
 		-subj "/C=DE/O=SAP SE/OU=testmachinery/CN=testmachinery-controller.default.svc"
 	@openssl x509 -req -sha256 -days 365 -in assets/tls.csr -CA assets/ca.crt -CAkey assets/ca.key -CAcreateserial -out assets/tls.crt
 
-.PHONY: create-ns
-create-ns:
-	@ if [ ${NS} != "default" ] && [ ! kubectl get ns ${NS} &> /dev/null ]; then \
-		kubectl create ns ${NS}; \
-	fi
 
-.PHONY: delete-ns
-delete-ns:
-	@ if [ ${NS} != "default" ] && [ kubectl get ns ${NS} ]; then \
-		kubectl delete ns ${NS}; \
-	fi
+################################
+# Prerequisistes, Installation #
+################################
+
+.PHONY: deploy-controller
+deploy-controller:
+	helm template --namespace ${NS} -f ./charts/testmachinery/local-values.yaml --set "controller.tag=${VERSION}" ./charts/testmachinery | kubectl create -f -
```

Comment:
Doesn't `kubectl apply` make more sense, as in e.g. deploying the controller subsequently after changing code?